Jakefile.js

```diff
@@ -100,11 +100,20 @@ var serverSources = [
     "editorServices.ts",
     "protocol.d.ts",
     "session.ts",
+    "nodeimpl.ts",

```

Comment:
i do not think you need the extra file if we pass the values as a constructor parameter.